_docs/style/text-conventions/external-internal-links.md

```diff
@@ -0,0 +1,52 @@
+---
+title: External and Internal Link Guidelines
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Use these guidelines when you construct links to external or internal content.
+resource: false
+status: complete
+last-modified: 2019-03-15
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-6-md hxSpan-6-lg" %}
+
+## General guidelines
+
+When you add links from a control page to internal or external content, consider the user's context and information needs when deciding whether to open the contents of a link in the current tab, or in another tab.
```

Comment:
Use Links style for any links in a control panel page. (xref typography)
Links are only to be use for navigation
Be sure you consider the users  context and information needs when deciding proper links behavior for your use case.

Reply:
Done!